wordpress-mobile/WordPress-iOS
# [Tooling] Update fastlane + remove workaround for match mutation

## What

 - Update `fastlane` to `2.205.2`
 - Which finally allows us to remove the workaround / patch we had to use because of `match` mutating one of its parameter — as my fix for it in https://github.com/fastlane/fastlane/pull/20125 has finally been released in `2.205.2` 🎉 

## To Test

 - Run `bundle exec fastlane update_certs_and_profiles` and verify it doesn't crash
